src/environments/array.js

```diff
@@ -573,8 +573,8 @@ const mathmlBuilder: MathMLBuilder<"array"> = function(group, options) {
     // We simulate this by adding (arraystretch - 1)em to the gap. This
     // does a reasonable job of adjusting arrays containing 1 em tall content.
 
-    // The 0.16 and 0.09 values are found emprically. They produce an array
-    // similar to LaTeX and in which content does not interfere with \hines.
+    // The 0.16 and 0.09 values are found empirically. They produce an array
+    // similar to LaTeX and in which content does not interfere with \hlines.
```

Comment:
I'm not sure what `\hines` is. I'm _guessing_ it's a typo. But I could be very wrong.

Reply:
It's a typo.